### DISCUSSION TITLE
Import cryoEM density and color by underlying model

### DISCUSSION THREAD

**Discussion by HicksChadW** [BradyAJohnston/MolecularNodes#165] [Ideas]
Hi, this suggestion is derived from a twitter reply chain and posted here to keep all suggestions in one place.

For an upcoming version of Molecular Nodes, Brady mentioned that Molecular Nodes will be able to upload cryoEM electron density to Blender. It would be great to have the functionality to color EM density by proximity using the color of an underlying atomic model. This is very similar to the ChimeraX command "color zone". It would really help establish Molecular Nodes as a suitable replacement to ChimeraX for making cryoEM structural figures. Thanks!

**Comment by BradyAJohnston** [comment]
Have already been testing [exactly this feature](https://twitter.com/bradyajohnston/status/1627285280775610369?s=20). Thanks for making it an official request though, this will help for me to keep track of it all :)

![image](https://user-images.githubusercontent.com/36021261/220212847-0a48cafe-fb0e-4854-a2ab-3dcec1c795c7.png)
